### PR TITLE
Get bbc news

### PR DESCRIPTION
**Pull request by KhushalPShah**
# Description

Added a script to get the latest news from the BBC News Web API.
It provides the headlines as well as the links to read the full article.

Fixes #(https://github.com/HarshCasper/Rotten-Scripts/issues/457)

## Type of change

_Please delete options that are not relevant._

- [*] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by KhushalPShah** [comment]
Can someone tell me why is the check failing ? 
PS : I am new to open source, and hence not sure of this issue.

Thanks!

File: Python/Get_BBC_News/getNews.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import requests
+
+def NewsFromBBC():
+     # BBC news api 
+    main_url = " https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=4dbc17e007ab436fb66416009dfb59a8"
```
Comment (HarshCasper) [inline Python/Get_BBC_News/getNews.py:5]: You should never be keeping your API Key on your Source Code. Create a `.env` file and make use of `Dotenv` module to pass secret information to your code. Make sure to add a `.env.example` to help other developers know that they can keep their secret keys into that.

File: Python/Get_BBC_News/getNews.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    open_bbc_page = requests.get(main_url).json()
+    articles = open_bbc_page['articles']
+    results = []
+    urls = []
+    
+    for ar in articles:
```
Comment (HarshCasper) [inline Python/Get_BBC_News/getNews.py:11]: Make use of meaningful variable names. `ar` doesn't seem to be consistent with any naming convention.

File: Python/Get_BBC_News/getNews.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        results.append(ar['title'])
+        urls.append(ar['url'])
+    
+    for i in range(len(results)):
+        print(i+1, results[i])
+        print("Link to the full article : " + urls[i] + '\n')
+        
+
```
Comment (HarshCasper) [inline Python/Get_BBC_News/getNews.py:19]: Remove these trailing lines.

File: Python/Get_BBC_News/getNews.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    
+if __name__ == "__main__":
+    NewsFromBBC()
```
Comment (HarshCasper) [inline Python/Get_BBC_News/getNews.py:22]: Run your Code through PyLint and see what improvements you can do depending on your score.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by KhushalPShah** [comment]
Have added a new commit.
Please check and let me know if there are some more things missing from my side.

Thanks @HarshCasper  for the help!

**Comment by KhushalPShah** [comment]
Have added a new commit.
Please check and let me know if there are some more things missing from my side.

Thanks @HarshCasper for the help!

**Comment by KhushalPShah** [comment]
Is there anything more to be done here @HarshCasper ?
If not, can you merge this?

**Comment by HarshCasper** [comment]
Great Work @KhushalPShah

**Pull request merged**